### PR TITLE
add PyGamer Thermal Camera code folder

### PR DESCRIPTION
**Pull request by CedarGroveStudios**
Create a folder and add code+font files for the PyGamer Thermal Camera learning guide.

### CODE REVIEW

File: PyGamer_Thermal_Camera/code.py [hunk 1]
```diff
@@ -0,0 +1,431 @@
...
+
+        # Adjust parameter value
+        param_value = int(image_group[param_index + 70].text)
+        while (not panel.button.a) and (not panel.button.start):
+            left, right, up, down = move_buttons(joystick=panel.has_joystick)
+            if up:
+                param_value = param_value + 1
+            if down:
+                param_value = param_value - 1
+            if param_value  > MAX_SENSOR_F:
```
Comment (ladyada) [inline PyGamer_Thermal_Camera/code.py]: could do param_value = max(MIN_SENSOR_F, min(MAX_SENSOR_F, param_value))

File: PyGamer_Thermal_Camera/code.py [hunk 2]
```diff
@@ -0,0 +1,431 @@
...
+    # Display average label and value
+    ave_label.color = YELLOW
+    ave_value.color = YELLOW
+    return int(alarm_value.text), int(max_value.text), int(min_value.text)
+
+def move_buttons(joystick=False):  # Read position buttons and joystick
+    move_r = move_l = False
+    move_u = move_d = False
+    if joystick:  # For PyGamer: interpret joystick as buttons
+        if   panel.joystick[0] > 44000:
```
Comment (ladyada) [inline PyGamer_Thermal_Camera/code.py]: did ya look at cursorcontrol? https://github.com/adafruit/Adafruit_CircuitPython_CursorControl/tree/master/adafruit_cursorcontrol

Comment (CedarGroveStudios) [reply PyGamer_Thermal_Camera/code.py]: Yes, but it was more complex than the simple up-down setup buttons that were needed for this project.

File: PyGamer_Thermal_Camera/code.py (hunk 1 above)
Comment (CedarGroveStudios) [reply PyGamer_Thermal_Camera/code.py]: Excellent simplification. Thank you!

Comment (CedarGroveStudios) [reply PyGamer_Thermal_Camera/code.py]: Could also do that at line 170, as well.

**Pull request closed without merging**